rmf_demo_maps/maps/office/office.building.yaml

```diff
@@ -1,9 +1,30 @@
 levels:
   L1:
+    crowd_sim:
+      agent_profiles:
+        - {ORCA_tau: 1, ORCA_tauObst: 0.4, class: 1, max_accel: 0, max_angle_vel: 0, max_neighbors: 10, max_speed: 0, name: external_agent, neighbor_dist: 5, obstacle_set: 1, pref_speed: 0, r: 0.25}
+        - {ORCA_tau: 1, ORCA_tauObst: 1, class: 1, max_accel: 3, max_angle_vel: 270, max_neighbors: 10, max_speed: 2, name: human, neighbor_dist: 5, obstacle_set: 1, pref_speed: 1.3, r: 0.18}
+      enable: 1
+      external_agent_groups:
+        - {agents_name: [tinyRobot2, tinyRobot1], agents_number: 2, group_id: 0, profile_selector: external_agent, state_selector: external_static, x: 0, y: 0}
+      goal_sets:
+        - {capacity: 2, set_area: [hardware_2], set_id: 0}
+        - {capacity: 4, set_area: [conf_1, conf_2], set_id: 1}
+      model_types:
+        - {animation: walk, animation_speed: 1, typename: human}
+      obstacle_set: {class: 1, file_name: navmesh.nav, type: nav_mesh}
+      states:
+        - {final: 1, goal_set: -1, name: external_static, navmesh_file_name: ""}
+        - {final: 0, goal_set: 0, name: thinking, navmesh_file_name: navmesh.nav}
+        - {final: 0, goal_set: 1, name: conf_runner, navmesh_file_name: navmesh.nav}
+      transitions:
+        - {Condition: {condition1: {distance: 0.8, type: goal_reached}, condition2: {dist: c, per_agent: true, type: timer, value: 20}, type: or}, Target: [{name: conf_runner, weight: 1}, {name: thinking, weight: 1}], from: thinking, to: ""}
+        - {Condition: {condition1: {distance: 0.8, type: goal_reached}, condition2: {dist: c, per_agent: true, type: timer, value: 20}, type: or}, Target: [{name: conf_runner, weight: 1}, {name: thinking, weight: 1}], from: conf_runner, to: ""}
+      update_time_step: 0.1
     doors:
-      - [31, 13, {motion_axis: [1, start], motion_degrees: [3, 90], motion_direction: [2, 1], name: [1, main_door], type: [1, double_hinged], plugin: [1, normal]}]
-      - [35, 37, {motion_axis: [1, start], motion_degrees: [3, 90], motion_direction: [2, 1], name: [1, coe_door], type: [1, hinged], plugin: [1, normal]}]
-      - [18, 17, {motion_axis: [1, start], motion_degrees: [3, 90], motion_direction: [2, 1], name: [1, hardware_door], type: [1, hinged], plugin: [1, normal]}]
+      - [31, 13, {motion_axis: [1, start], motion_degrees: [3, 90], motion_direction: [2, 1], name: [1, main_door], plugin: [1, normal], type: [1, double_hinged]}]
+      - [35, 37, {motion_axis: [1, start], motion_degrees: [3, 90], motion_direction: [2, 1], name: [1, coe_door], plugin: [1, normal], type: [1, hinged]}]
+      - [18, 17, {motion_axis: [1, start], motion_degrees: [3, 90], motion_direction: [2, 1], name: [1, hardware_door], plugin: [1, normal], type: [1, hinged]}]
```

Comment:
This change also seems unnecessary?

Reply:
this change is auto generated upon save in traffic editor, essentially just some random reordering, which is deemed unnecessary.